templates/gaming/index.html

```diff
@@ -0,0 +1,203 @@
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6">
+      <h2>What’s under the hood</h2>
+      <h4>Anbox Cloud builds on a powerful stack of open-source software to deliver Android game streaming from the cloud.</h4>
+    </div>
+  </div>
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h3>Android in containers</h3>
+      <p>Unlike Android emulators that make use of virtual machines, Anbox Cloud leverages system containers through <a class="p-link--external" href="https://linuxcontainers.org/">LXD</a>, the next generation system container manager. It offers a user experience similar to virtual machines but using lightweight Linux containers instead manageable via a REST API. It also exposes GPUs through a passthrough. You will therefore take advantage of any hardware accelerators to deliver rich mobile experiences.</p>
+    </div>
+    <div class="col-5 u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/ac9a0e00-lxd_primary.svg",
+        alt="",
+        width="229",
+        height="75",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h3>Software orchestration</h3>
+      <p>Streaming Android apps to thousands of mobile and desktop clients simultaneously requires orchestration and monitoring of several software components. This involves streaming servers, image repositories, applications, a dashboard application for management and <a href="https://anbox-cloud.io/docs/overview">more</a>. Thanks to <a href="https://jaas.ai/">Juju</a> and <a href="https://maas.io/">MAAS</a>, you can deploy your stack automatically anywhere.</p>
+    </div>
+    <div class="col-5 u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/48a8eddf-juju_black-orange_hex.svg",
+        alt="",
+        width="208",
+        height="75",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+        }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-vertically-center">
+    <div class="col-6 u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
+        alt="",
+        width="289",
+        height="205",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+    <div class="col-5">
+      <h3>Stream Android games</h3>
+      <p>Get in touch with Canonical to explore avenues to deliver your games to millions via cloud streaming. </p>
+      <button href="/#request-form"  class="p-button--positive">Get in touch</button>
```

Comment:
Sorry it looks like there is an contact us page: https://anbox-cloud.io/contact-us